index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+  <title>Document</title>
+</head>
+<body>
+  
+  <header>
+    <!-- <img src="./images/background/all_shapes.svg" background="" alt="geometric figures in the background"> -->
+    <div class="header-top">
+      <div class="logo">
+        <p>
+          My Logo
+        </p>
+      </div>
+      <div class="menu">
+        <img src="./images/header/burger_menu.png" alt="menu button">
+      </div>
+    </div>
+    <div class="header-bottom">
+      <div class="greeting">
+        <p>I'm Christian <br>
+          Glad to see you!
+        </p>
+      </div>
+      <div class="presentation">
+        <p>
+          I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+        </p>
+      </div>
+      <div class="connect">
+        <p>LET'S CONNECT</p>
+      </div>
+      <div class="media-icons">
+        <div class="icons">
+          <img src="./images/media_icons/twitter.svg" alt="twitter icon">
+        </div>
+        <div class="icons">
+          <img src="./images/media_icons/linkedin.svg" alt="linkedin icon">
+        </div>
+        <div class="icons">
+          <img src="./images/media_icons/mail.svg" alt="mail icon">
+        </div>
+        <div class="icons">
+          <img src="./images/media_icons/github.svg" alt="github icon">
+        </div>
+        <div class="icons">
+          <img src="./images/media_icons/hand.svg" alt="other icon">
+        </div>
```

Comment:
- [ ] nice job. please your list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.